enchant2.yaml

```
@@ -28,7 +28,9 @@ pipeline:
     with:
       repository: https://github.com/rrthomas/enchant
       tag: v${{package.version}}
-      expected-commit: 2e2739acea4926ae2584823b5f75480243b191df
+      expected-commit: 80115e50c244f2dde3da41b8387e6e7b2ff65f52
+
+  - runs: git submodule update --init
```

IIUC, `git-checkout` can handle this for us:
```suggestion
      expected-commit: 80115e50c244f2dde3da41b8387e6e7b2ff65f52
      recurse-submodules: true
```